ckanext/noanonaccess/plugin.py

```diff
@@ -19,6 +19,8 @@ def __call__(self, environ, start_response):
             return self.app(environ,start_response)
         elif '/uploads/' in environ['PATH_INFO'] or '/download/' in environ['PATH_INFO']:
             return self.app(environ,start_response)
+        elif '.rdf' in environ['PATH_INFO']:
```

Comment:
@gavram can we do "ends with" instead of "in"?

Reply:
Yes we can. Will do it.

Reply:
@anuveyatsu 

Done